configure

```diff
@@ -856,230 +856,6 @@ then
     putvar CFG_LOCAL_RUST_ROOT
 fi
 
-# Force bitrig to build with clang; gcc doesn't like us there
```

Comment:
In the future, it's useful for large code movement like this to be split out into a separate commit so the changes to it can be reviewed separately, but it's fine for now